src/main/java/uk/gov/hmcts/reform/sendletter/controllers/SendLetterController.java

```diff
@@ -91,7 +91,7 @@ public SendLetterController(
         @RequestHeader("ServiceAuthorization") String serviceAuthHeader
     ) {
         tokenValidator.getServiceName(serviceAuthHeader); //TODO: check that this service is allowed to do it
-        letterService.updateSentToPrintAt(id, patch);
+        letterService.updateSentToPrintAt(getLetterIdFromString(id), patch);
```

Comment:
might want to get ID first thing so to avoid extra http calls for token validation

Reply:
sending 404 when 'user' is not authenticated would be weird :)